roles/common/tasks/main.yml

```diff
@@ -4,12 +4,10 @@
     hostname "{{inventory_hostname}}"
     echo "{{inventory_hostname}}" > /etc/hostname
   when: ansible_hostname != inventory_hostname
-- name: gather facts again
-  setup:
-- name: Build hosts file
-  lineinfile: dest=/etc/hosts regexp='.*{{ item }}$' line="{{ hostvars[item].ansible_default_ipv4.address }} {{item}}" state=present
-  when: hostvars[item].ansible_default_ipv4 is defined and hostvars[item].ansible_default_ipv4.address is defined
-  with_items: "{{ groups['all'] }}"
+- name: improve log timestamps
+  shell: |
+    sed -i 's/RSYSLOG_TraditionalFileFormat/RSYSLOG_FileFormat/' /etc/rsyslog.conf
+    systemctl restart rsyslog
```

Comment:
I think the ansible way of doing this would be to use [lineinfile](http://docs.ansible.com/ansible/latest/lineinfile_module.html#examples) and use the first part of the regexp to match the line you want to change and then write the full line how it should like (or you can use `\1` in it). And the restart would be a `notify` so syslog is only restarted if the line actually had to be changed.

Reply:
Thanks for the tip! I have changed this in the replacement commit below.